documentation/dotnet-dump-instructions.md

```diff
@@ -19,7 +20,7 @@ If this is the first global tool installed or you get message `Could not execute
 
 ## Using dotnet-dump
 
-The next step is to collect a dump. This can be skipped if a core dump has already been generated by the operating system or [createdump](https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/xplat-minidump-generation.md#configurationpolicy) on Linux.
+The next step is to collect a dump. This can be skipped if a core dump has already been generated by the operating system or [createdump](https://github.com/dotnet/runtime/blob/master/docs/design/coreclr/botr/xplat-minidump-generation.md#configurationpolicy) on Linux. The default dump type (--type option) is currently "full".
```

Comment:
Maybe add a cautionary note about the level required in core_filter for OS dumps

Reply:
coredump_filter is documented in the debugging-coredump.md and in the FAQ.md.